README.md

```diff
@@ -74,8 +74,93 @@ require_once __DIR__ . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'a
 $runner->addExtension(new \mageekguy\atoum\blackfire\extension());
 ```
 
+## Other examples
+
+### Define custom metrics
+
+```php
+$this
+    ->blackfire()
+        ->defineMetric(new \Blackfire\Profile\Metric("example_method_calls", "=Example::method"))
```

Comment:
Why the user has to create a `Backfire\Profile\Metric` object? Can it be another kind of object?